client/state/jetpack-connect/actions/check-url.js

```diff
@@ -23,7 +23,7 @@ import 'state/jetpack-connect/init';
 const _fetching = {};
 const debug = debugFactory( 'calypso:jetpack-connect:actions' );
 
-export function checkUrl( url, isUrlOnSites ) {
+export function checkUrl( url, isUrlOnSites, isPurchasingSearch ) {
```

Comment:
I feel like the name here should just be "allowWPCOMSite" that way if there are other cases in the future it can support those as well.